src/doc/guide.md

```diff
@@ -746,11 +736,7 @@ fn print_sum(x: int, y: int) {
 }
 ```
 
-You separate arguments with a comma, both when you call the function, as well
-as when you declare it.
```

Comment:
I suppose I thought it was trivial. Now that I think about it, it's not, but I ithink it can be worked in more succinctly to the prior paragraph.

I'll see what I can come up with...